lib/node.py

```diff
@@ -126,7 +126,9 @@ def docker_inspect(self, container_name, output_format=None):
     def docker_exec(self, container_name, cmd):
         command = 'docker exec {0} {1}'.format(container_name, cmd)
         result = self.execute_command(command)
-        if result[1] != '':
```

Comment:
Why do we not want to treat warning shown in stderror as an exception in this case ?

Reply:
This is to handle the case where etcdctl issues warning when the cluster health is healthy for some k8s versions. If we don't handle this case, then exception will be raised even when the cluster is healthy for some k8s versions.